src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarWriter.Windows.cs

```diff
@@ -54,7 +54,7 @@ private TarEntry ConstructEntryForWriting(string fullPath, string entryName, Fil
             entry._header._aTime = info.LastAccessTimeUtc;
             entry._header._cTime = info.LastWriteTimeUtc; // There is no "change time" property
 
-            entry.Mode = DefaultWindowsMode;
+            entry.Mode = TarHelpers.DefaultWindowsMode;
```

Comment:
Two windows build failures due to DefaultWindowsMode existing inside the TarWriter class, not in TarHelpers:

```suggestion
            entry.Mode = DefaultWindowsMode;
```

Reply:
@tmds I verified in my local machine that this was the correct change to get rid of the build failure. I'll apply it as a suggestion so that we can get the test results ASAP (the RC1 snap will happen on Thursday).

Let me know if you did not intend to have the DefaultWindowsMode field defined inside TarWriter.

Reply:
This looks good. Thanks.